packages/gatsby-source-prismic/src/create-schema-customization.ts

```diff
@@ -1,28 +1,24 @@
-import * as gatsby from 'gatsby'
-import * as RTE from 'fp-ts/ReaderTaskEither'
-import * as R from 'fp-ts/Record'
 import * as A from 'fp-ts/Array'
 import * as E from 'fp-ts/Either'
-import { pipe, constVoid } from 'fp-ts/function'
-
-import { createAllDocumentTypesType } from './lib/createAllDocumentTypesType'
-import { createCustomType } from './lib/createCustomType'
-import { createTypes } from './lib/createTypes'
-import { throwError } from './lib/throwError'
-
+import { constVoid, pipe } from 'fp-ts/function'
+import * as RTE from 'fp-ts/ReaderTaskEither'
+import * as R from 'fp-ts/Record'
+import * as gatsby from 'gatsby'
+import { buildDependencies } from './buildDependencies'
 import { buildEmbedType } from './builders/buildEmbedType'
 import { buildGeoPointType } from './builders/buildGeoPointType'
 import { buildImageDimensionsType } from './builders/buildImageDimensionsType'
 import { buildImageThumbnailType } from './builders/buildImageThumbnailType'
-import { buildImgixImageTypes } from './builders/buildImgixImageTypes'
 import { buildLinkType } from './builders/buildLinkType'
 import { buildLinkTypeEnumType } from './builders/buildLinkTypeEnumType'
 import { buildSliceInterface } from './builders/buildSliceInterface'
 import { buildStructuredTextType } from './builders/buildStructuredTextType'
 import { buildTypePathType } from './builders/buildTypePathType'
-
+import { createAllDocumentTypesType } from './lib/createAllDocumentTypesType'
+import { createCustomType } from './lib/createCustomType'
+import { createTypes } from './lib/createTypes'
+import { throwError } from './lib/throwError'
 import { Dependencies, Mutable, PluginOptions } from './types'
-import { buildDependencies } from './buildDependencies'
```

Comment:
Is your editor sorting imports? Is it possible to leave these changes out of the PR?

Reply:
Yeah, my editor sorts imports. I can back these out manually.

Reply:
Thank you! This is one of those things where I wish Prettier would automatically sort these things out in a logical way, like Go's formatter.